src/Bicep.LangServer.UnitTests/BicepCompilationManagerHelper.cs

```diff
@@ -47,6 +43,21 @@ public static BicepCompilationManager CreateCompilationManager(DocumentUri docum
             return bicepCompilationManager;
         }
 
+        public static BicepCompilationManager CreateCompilationManager(ILanguageServerFacade server, IWorkspace workspace)
+        {
+            var helper = TestDiHelper.Create(services => services
+                .AddSingleton<ILanguageServerFacade>(server)
+                .AddSingleton<IAzResourceTypeLoader>(TestTypeHelper.CreateEmptyAzResourceTypeLoader())
+                .AddSingleton(CreateMockScheduler().Object)
+                .AddSingleton(BicepTestConstants.CreateMockTelemetryProvider().Object)
+                .AddSingleton<ICompilationProvider, BicepCompilationProvider>()
+                .AddSingleton<IWorkspace>(workspace)
+                .WithFeatureOverrides(new())
```

Comment:
Is this line needed?

Reply:
Yes, it sets the assembly version to "dev", which is necessary for a number of the tests to pass. I've added a comment to explain.